fixtures/media-chrome/index.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css"
+    >
+    <style>
+      media-controller {
+        display: block;
+        width: 100%;
+        aspect-ratio: 16 / 9;
+        background: #000;
+      }
+    </style>
+  </head>
+  <body>
+    <script async src="https://cdn.jsdelivr.net/npm/es-module-shims"></script>
+    <script type="importmap">
+      {
+        "imports": {
+          "super-media-element": "https://cdn.jsdelivr.net/npm/super-media-element@1.3/+esm",
+          "media-tracks": "https://cdn.jsdelivr.net/npm/media-tracks@0.2/+esm",
+          "@superstreamer/player": "/packages/player/dist/index.js",
+          "hls.js": "https://cdn.jsdelivr.net/npm/hls.js@1.6.0-beta.1/dist/hls.mjs"
+        }
+      }
+    </script>
+    <script type="module" src="./superstreamer-video-element.js"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome/+esm"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome/menu/+esm"></script>
+
+    <media-controller>
+      <superstreamer-video  
+        slot="media"
+        src="https://stitcher.superstreamer.xyz/session/8121e4cc-2024-45b5-bfc8-323dec368819/master.m3u8">
+      </superstreamer-video>
+      <media-loading-indicator slot="centered-chrome" no-auto-hide></media-loading-indicator>
```

Comment:
```suggestion
      <media-loading-indicator slot="centered-chrome" noautohide></media-loading-indicator>
```

I was wondering where you got this markup? seems to have the old way of kebab cased attributes.
the new attributes in Media Chrome are all without dashes in them.